src/client/phy-ue.cc

```diff
@@ -121,6 +130,20 @@ PhyUe::PhyUe(Config* config)
     workers_.push_back(std::move(new_worker));
   }
 
+  if (kRecordDownlinkFrame) {
+    //Add and writter / record type here
+    std::vector<Agora_recorder::RecorderWorker::RecorderWorkerTypes> recorders;
+    recorders.push_back(Agora_recorder::RecorderWorker::RecorderWorkerTypes::
+                            kRecorderWorkerHdf5);
```

Comment:
Maybe define the worker type as a constexpr at the top, like:
#if defined(ENABLE_HDF5)
constexpr kRecorderType = ...kRecorderWorkerHdf5
#else
constexpr kRecorderType = ...kRecorderWorkerMultiFile

Reply:
t is really a vector.  I moved it to the top